addons/payment_adyen/__init__.py

```diff
@@ -3,3 +3,10 @@
 
 from . import models
 from . import controllers
+
+from odoo import api, SUPERUSER_ID
+
+
+def _auto_create_journal(cr, registry):
```

Comment:
factorize in payment
import odoo.addons.payment import create_missing_journal_for_aqurier